src/cpp/main.cpp

```diff
@@ -47,32 +51,48 @@ size_t initialize(const Config *config) {
   displacements = (Vector*) calloc(num_particles, sizeof(Vector));
 
   // Initialize the particles.
+
+  // Particles radious generator in milimeters
+  int seed = 7;
```

Comment:
The seed should come from the input file. 😉

Reply:
Ok I'll add that.